dm/config/task.go

```diff
@@ -266,9 +266,10 @@ type LoaderConfig struct {
 // DefaultLoaderConfig return default loader config for task.
 func DefaultLoaderConfig() LoaderConfig {
 	return LoaderConfig{
-		PoolSize:   defaultPoolSize,
-		Dir:        defaultDir,
-		ImportMode: LoadModeSQL,
+		PoolSize:    defaultPoolSize,
+		Dir:         defaultDir,
+		ImportMode:  LoadModeSQL,
+		OnDuplicate: OnDuplicateReplace,
```

Comment:
Can we add a test case with minimal job config to test all default value? Maybe add in unit test like what we do in job_template

Reply:
going to modify dm-job.yaml in current e2e test